static/bidder-params/ix.json

```diff
@@ -6,8 +6,18 @@
   "properties": {
     "siteId": {
       "type": "string",
+      "minLength": 1,
       "description": "An ID which identifies the site selling the impression"
+    },
+    "size": {
+      "type": "array",
+      "items": {
+        "type": "integer"
+      },
+      "minItems": 2,
+      "maxItems": 2,
+      "description": "An array of two integer containing the dimension"
     }
   },
-  "required": ["siteId"]
+  "required": ["siteId", "size"]
```

Comment:
Bidder params can't have breaking changes. Since they're in Stored Requests, they may be in host companies' DBs. It's unrealistic to make host companies do DB migrations, at least not without a significant breaking change window.

Can you add this as an optional param, and make your adapter return `BadInput` errors on requests that don't have it?